web/app/view/ServerDialog.js

```diff
@@ -66,24 +66,45 @@ Ext.define('Traccar.view.ServerDialog', {
             displayField: 'name',
             valueField: 'key'
         }, {
-            xtype: 'numberfield',
-            name: 'latitude',
-            fieldLabel: Strings.positionLatitude,
-            decimalPrecision: Traccar.Style.coordinatePrecision
-        }, {
-            xtype: 'numberfield',
-            name: 'longitude',
-            fieldLabel: Strings.positionLongitude,
-            decimalPrecision: Traccar.Style.coordinatePrecision
-        }, {
-            xtype: 'numberfield',
-            name: 'zoom',
-            fieldLabel: Strings.serverZoom
+            xtype: 'fieldcontainer',
```

Comment:
Is it OK if I continue using `this.getView().down('numberfield[name="latitude"]')` to access values or it is better to add `reference` and use `lookupReference()` ?